pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
```

Comment:
What's the benefit of having a `pyproject.toml`? Looks like it's not adding anything important. Rather adding more files and config and therefor more things that need maintenance.